src/library/baseexternallibraryfeature.h

```diff
@@ -26,9 +27,10 @@ class BaseExternalLibraryFeature : public LibraryFeature {
 
   protected:
     // Must be implemented by external Libraries copied to Mixxx DB
-    virtual BaseSqlTableModel* getPlaylistModelForPlaylist(const QString& playlist) {
+    virtual std::unique_ptr<BaseSqlTableModel> createPlaylistModelForPlaylist(
+            const QString& playlist) {
```

Comment:
prefer `QStringView` over `const QString&` afaik. 
```suggestion
    virtual std::unique_ptr<BaseSqlTableModel> createPlaylistModelForPlaylist(
            QStringView playlist) {
```

Reply:
I have just tried that, but it fails, because of the final `QVariant(const QString&)` call. 
This requires a QStringView::toString() which performs a deep copy.

Reply:
Mhmm shame, I guess replacing that with a `QVariant(QStringView)` is not possible?